frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -3987,7 +4010,7 @@ def zeros(self, resulting_type: ConversionType) -> Conversion:
         )
 
     def get_partition_name(self, partition: tfhers.TFHERSParams) -> str:
-        if partition not in self.tfhers_partition.keys():
+        if partition not in self.tfhers_partition:
```

Comment:
that should not part of this PR? (cc @youben11)

Reply:
Ah yeah, but no worries if you want to merge it as I didn't fix it on my side yet

Reply:
Sure, I'll rollback the change on this PR, but another PR should fix it.